Looks like new type of [failure](https://jenkins.dockerproject.org/job/Docker-PRs/54812/console) to me in the janky PR check:
```
--- FAIL: Test (3896.19s)
panic: DockerSwarmSuite.TestSwarmJoinPromoteLocked test timed out after 5m0s [recovered]
	panic: DockerSwarmSuite.TestSwarmJoinPromoteLocked test timed out after 5m0s
```

--- a/internal/test/daemon/daemon.go
+++ b/internal/test/daemon/daemon.go
@@ -722,9 +722,11 @@ func cleanupRaftDir(t testingT, rootPath string) {
 	if ht, ok := t.(test.HelperT); ok {
 		ht.Helper()
 	}
-	walDir := filepath.Join(rootPath, "swarm/raft/wal")
-	if err := os.RemoveAll(walDir); err != nil {
-		t.Logf("error removing %v: %v", walDir, err)
+	for _, p := range []string{"wal", "wal-v3-encrypted", "snap-v3-encrypted"} {
+		dir := filepath.Join(rootPath, "swarm/raft", p)
+		if err := os.RemoveAll(dir); err != nil {
+			t.Logf("error removing %v: %v", dir, err)
+		}
 	}
 }
 

